doc/man/scons.xml

```diff
@@ -2156,8 +2177,57 @@ repositories are searched in the order specified.</para>
 
 <refsect1 id='configuration_file_reference'>
 <title>CONFIGURATION FILE REFERENCE</title>
-<!--  .SS Python Basics -->
-<!--  XXX Adding this in the future would be a help. -->
+
+<refsect2 id='configuration_files'>
+<title>Configuration Files</title>
+<para>
+The build configuration consists of one or more configuration files
+(SConscript files).
+The main file is named <filename>SConstruct</filename> by
+default, though if necessary, &scons; also checks for alternate spellings
```

Comment:
alternative filenames instead of spellings.

Reply:
I'll make this change.